scripts/CSSRE/get-node-restarts/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+file: script.py
+name: get-node-restarts
+description: Script to get list of nodes that are restarted in the last X minutes
+author: vbommana
+allowedGroups: 
+  - CSSRE
+rbac:
+  clusterRoleRules:
+    - verbs:
+      - "get"
+      - "list"
+      apiGroups:
+      - ""
+      resources:
+      - "nodes"
+      - "pods"
```

Comment:
Do we need these perms, by default the SA used by the pod  already has cluster-reader permissions

Reply:
Hi @wanghaoran1988 , I've tried removing the rbac `clusterRoleRules` from `metadata.yaml` and tested in staging..
I am getting the following error:
```
2022-03-11 00:12:23,248.248 ERROR (py.script):  Error from server (Forbidden): nodes is forbidden: User "system:serviceaccount:openshift-backplane-managed-scripts:openshift-job-dev-r9phq" cannot list resource "nodes" in API group "" at the cluster scope
```

```yaml
-rbac:
-  clusterRoleRules:
-    - verbs:
-      - "get"
-      - "list"
-      apiGroups:
-      - ""
-      resources:
-      - "nodes"
-      - "pods"
```
may I shouldn't remove entire rbac ?

Reply:
I may wrong, let's keep this.

Reply:
ok